happytransformer/qa/trainer.py

```diff
@@ -54,18 +54,11 @@ def test(self, input_filepath, solve):
         """
         contexts, questions = self._get_data(input_filepath, test_data=True)
 
-        results = list()
-
-        for case in tqdm(zip(contexts, questions)):
-            context = case[0]
-            question = case[1]
-            result = solve(context, question)[0]  # only care about first result
-
-            results.append(result)
-
-        return results
-
-
+        return [
+            solve(context,question)[0]
```

Comment:
Awkward, the original way is easier to understand

Reply:
I disagree. This is a perfect case for a list comprehension

Reply:
It's no big deal, we can keep it as is then. I just think it looks awkward with the tqdm in the mix